.github/workflows/reusable-integration-tests-on-prem-nightly.yml

```
@@ -39,6 +39,7 @@ jobs:
 
     strategy:
       matrix:
+        node: [ "lts/*", 19 ]
```

I think it will be more convenient to test against the latest Node.JS build so that we could detect compatibility issues earliest possible.
```suggestion
        node: [ "lts/*", "latest"]
```

---

Yes good one!